install/elabftw.sql

```diff
@@ -391,11 +392,11 @@ INSERT INTO `items_types` (`team`, `id`, `name`, `color`, `template`) VALUES
 (1, 1, 'Edit me', '32a100', '<p>Go to the admin panel to edit/add more items types!</p>');
 
 /* the default status */
-INSERT INTO `status` (`team`, `id`, `name`, `color`, `is_default`) VALUES
-(1, 1, 'Running', '29AEB9', 1),
-(1, 2, 'Success', '54aa08', 0),
-(1, 3, 'Need to be redone', 'c0c0c0', 0),
-(1, 4, 'Fail', 'c24f3d', 0);
+INSERT INTO `status` (`team`, `id`, `name`, `color`, `allow_timestamp`, `is_default`) VALUES
```

Comment:
Instead of `allow_timestamp` it would be better to name this `is_timestampable`. It's more homogeneus with the `is_default` column.